wordpress-mobile/WordPress-iOS
# Jetpack App: Update App Settings and About screens

Resolves #16197

## Description
This PR updates app specific info under App Setting and About in the Jetpack app
- App Settings
  - Updated "About WordPress for iOS" to "About Jetpack for iOS" 
- About 
  -  Updated app icon (placeholder icon for now) 🚧 
  -  Updated Twitter handle to `@jetpack`
  -  Updated blog url to `https://jetpack.com/blog/`

App Settings | About | Twitter | Blog 
--- | --- | --- | ---
![Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 15 46 01](https://user-images.githubusercontent.com/6711616/113255874-54185d80-9303-11eb-8a3a-d798414bb1b2.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 15 49 42](https://user-images.githubusercontent.com/6711616/113255875-55498a80-9303-11eb-9f25-006b185ebbe6.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 15 50 18](https://user-images.githubusercontent.com/6711616/113255876-55e22100-9303-11eb-9460-12ef50559901.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 15 50 38](https://user-images.githubusercontent.com/6711616/113255887-58dd1180-9303-11eb-96ec-ea26ee8cbd23.png)
![Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 15 46 07](https://user-images.githubusercontent.com/6711616/113256451-297ad480-9304-11eb-97a6-87ebf0dfed4b.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 15 49 49](https://user-images.githubusercontent.com/6711616/113256456-2a136b00-9304-11eb-9245-65b91d00d8e9.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 15 50 26](https://user-images.githubusercontent.com/6711616/113256459-2aac0180-9304-11eb-8e8c-0d95e4707894.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 15 56 35](https://user-images.githubusercontent.com/6711616/113256466-2d0e5b80-9304-11eb-98c9-339e9e481312.png)

## How to test

Follow the same steps for both the WordPress and Jetpack target:

1. Go to My Site > Me > App Settings and check the Other section
    - WP: "About WordPress for iOS"
    - Jetpack: "About Jetpack for iOS"
2. Tap on "About ... for iOS"
    - WP: WordPress icon, twitter handle is `@WordPressiOS`, blog is `blog.wordpress.com`
    - Jetpack: "About Jetpack for iOS", twitter handle is `@jetpack`, blog is `jetpack.com`
3. Tap on the Twitter row item
    - WP: Opens the WordPress for iOS twitter
    - Jetpack: Opens the Jetpack twitter
4. Go back the About screen
5. Tap on the Blog row item 
    - WP: Opens the WordPress blog
    - Jetpack: Opens the Jetpack blog

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested on both Jetpack and WordPress

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
